bindings/cs/rl.net.native/rl.net.live_model.h

```diff
@@ -2,11 +2,14 @@
 
 #include "rl.net.native.h"
 
-typedef void (*managed_callback_t)(const reinforcement_learning::api_status&);
+typedef void (*background_error_callback_t)(const reinforcement_learning::api_status&);
+typedef void (*trace_logger_t)(int log_level, const char* msg);
 
 typedef struct livemodel_context {
     reinforcement_learning::live_model* livemodel;
-    managed_callback_t callback;
+    background_error_callback_t background_error_callback;
+    trace_logger_t trace_logger_callback;
```

Comment:
>trace_logger_t [](start = 4, length = 14)

[nit] to keep alignment with "background_error_callback_t" this should be "trace_logger_callback_t"

Reply:
fixed.